tremor-script/eqc/model.erl

```diff
@@ -35,6 +32,56 @@ combine_values(Key, SpecVal = #{}, Acc) ->
 combine_values(Key, SpecVal, Acc) ->
     maps:put(Key, SpecVal, Acc).
 
+patch_operation({insert, Key, Value}, Acc) ->
+    maps:put(Key, Value, Acc);
+patch_operation({merge, Key, Value}, Acc) ->
+    maps:fold(fun combine_values/3, #{Key => Value}, Acc);
+patch_operation({upsert, Key, Value}, Acc) ->
+    % does what we expect from upsert
+    maps:put(Key, Value, Acc);
+patch_operation({erase, Key}, Acc) ->
+    maps:remove(Key, Acc).
+
+-spec ast_eval(#vars{}, {}) -> {#vars{},
+				integer() | float() | boolean() | binary()}.
+
+% Test cases the patch eval function generates
```

Comment:
this documentation is in the wrong place

Reply:
Yeah, I kept it near patch_operation code